csharp/PhoneNumbers/PhoneNumberOfflineGeocoder.cs

```diff
@@ -162,6 +162,17 @@ internal PhoneNumberOfflineGeocoder(string phonePrefixDataDirectory, Assembly as
             return files;
         }
 
+        private static string[] ParseNameFromArchive(ZipArchiveEntry entry)
+        {
+            const int expectedLength = 2;
+            var name = entry.FullName.Split('.')[0].Split(Path.DirectorySeparatorChar);

```

Comment:
Can you just do a `.Split(Path.DirectorySeparatorChar, Path.AlthDirectorySeparatorChar)`?

Reply:
Also needs to be verified to also work with the AppVeyor Windows build CI. On the one hand it would be nice to port that job to also use GHAs but building on Windows allows for running unit tests against netframework.

Reply:
does the existing pull request check cover passing on the AppVeyor build? - https://ci.appveyor.com/project/twcclegg/libphonenumber-csharp/builds/48607119

Otherwise, i have also created new github actions to cover both windows and linux - https://github.com/twcclegg/libphonenumber-csharp/pull/204

Reply:
thanks for the suggestion, i've updated the code as recommended in this commit - https://github.com/twcclegg/libphonenumber-csharp/pull/202/commits/499a2fc4e6bbf1cd5eb1571acf07319a719e4dc9